src/server/ua_server_ns0_gds.c

```diff
@@ -0,0 +1,1338 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2024 (c) Fraunhofer IOSB (Author: Noel Graf)
+ */
+
+#include "ua_server_internal.h"
+#include <open62541/plugin/log_stdout.h>
```

Comment:
Don’t include the logger.
The core library never imports a specific plugin.
Rather use the logger from the configuration.